f8a_worker/workers/user_notifier.py

```diff
@@ -0,0 +1,69 @@
+"""Output: TBD."""
+
+import json
+
+from f8a_worker.base import BaseTask
+from f8a_worker.utils import get_session_retry
+
+
+class UserNotificationTask(BaseTask):
+    """Generates report containing descriptive data for dependencies."""
+
+    def get_dependency_data(self, github_repo):
+        """Fetch data for all linked dependencies of a github repository.
+
+        :param github_repo:
```

Comment:
Minor Nitpick, would be good to document here whether this requires repo URL or repo name.